CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributuing
+
+Contributions to this project are welcome, whether minor updates to documentation or entirely new features.
+This document outlines the technical and procedural requirements for your work to be included.
+All contributions, in any form, must also confirm with the [code of conduct](CODE_OF_CONDUCT.md).
+
+> :warning: This document is a living document, and will adapt based on the project's needs.
+> If some part of this document creates a hindrance, raise an issue explaining the problem.
+
+## Scope: What Lives in this Repository?
+
+MewBot's core design includes the tenant of modularity.
+The core repo will only accept features and components that do not add additional dependencies.
+
+When building new components, whether they are input/output modules or utilities for use in behaviours,
+consider if they could be a stand-alone repository and package.
+The MewBot team is happy to help set up these repositories in the GitHub group.
+
+## Projects and Issues: What Needs Doing?
+
+Issues are a fantastic way of keeping track of what work is needed and who is planning to do it.
+You can find all the current initiatives in the projects tab; these show the current areas of focus.
+If you are looking for a way to contribute to the project, this is a great place to start.
+
+Issues are also a place to plan how to approach future work.
+Openly discussing ideas allows us to find places where multiple people are looking at
+similar problems and ensure that we don't have different solutions.
```

Comment:
Suggest adding something on the lines of "Don't be shy in adding issues. All concerns, or thoughts, are wlecome. But please don't be offended if issues get merged as we work towards a solution - which may solve several of them at once."

Reply:
Or something on these lines more grammerly acceptable? As opening an issue can be fairly intimidating. I'm thinking of the eventual target audience - who, hopefully, will include quite a lot of low or no code individuals.

Reply:
The rewording is much better!